src/index.html

```diff
@@ -1,13 +1,423 @@
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#aboutUs" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#compareBikes" class="menu__link">Compare Bikes</a>
+            </li>
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contactUs" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--action"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section
+        class="page__section about"
+        id="aboutUs"
+      >
+        <div class="container">
+          <div class="grid grid--tablet">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">
+                Move Free
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--5-12">
+              <p class="about__text">
+                Shift your ride, not gears. Find the fastest
+                way to move in the city as the bike adapts intuitively
+                to power the speed you need.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section
+        class="page__section compare"
+        id="compareBikes"
+      >
+        <div class="container">
+          <h2 class="page__section-title">
+            Compare bikes
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="compare__item grid__item grid__item--1-4">
+              <article
+                class="card"
+              >
+                <div class="card__image-container">
+                  <img
+                    class="card__image"
+                    src="images/photos/sporty4.jpg"
+                    alt="sporty4"
+                  >
+                </div>
+                <h3 class="card__title">Sporty 4</h3>
+                <p class="card__paragraph">
+                  The iconic frame brought to a new performance
+                  height as a sporty, active ride.
+                </p>
+                <span class="card__price">$ 2 590</span>
+              </article>
+            </div>
+
+            <div class="compare__item grid__item grid__item--5-8">
+              <article class="card">
+                <div class="card__image-container">
+                  <img
+                    class="card__image"
+                    src="images/photos/rideInTownST.jpg"
```

Comment:
The alt text for the image should be more descriptive, like 'Ride in town ST bike'